### ISSUE TITLE
[Feature suggestion] Show notification when clipboard monitoring paused / started

### ISSUE THREAD

**Issue by Senliast** [clipto-pro/Desktop#108]
You can pause and resume clipboard monitoring on desktop platforms with Ctrl + Shift + P. Could you also add a notification, like on Android, that the clipboard monitoring is paused / started?

**Comment by atrashler** [comment]
> You can pause and resume clipboard monitoring on desktop platforms with Ctrl + Shift + P. Could you also add a notification, like on Android, that the clipboard monitoring is paused / started?

Now the app displays notification when you pause/resume the clipboard. I suppose that on your platforms it does not work. Could you please write:

1) your platform
2) does the app display any other notifications while it is working

**Comment by Senliast** [comment]
yeah, i will specify what i mean. So i use the app on Windows and macOS, on both when i press Ctrl + Shift + P, the clipboard will be pause or resumed, but no notification will be shown. And as i remember, the app didnt ever show any other notifications. I just would like to see notification to be sure, that the clipboard is disabled. I need it when copying passwords, as i dont want them to be captured. And it would be more quick as clicking the tray icon every time.

**Comment by atrashler** [comment]
> yeah, i will specify what i mean. So i use the app on Windows and macOS, on both when i press Ctrl + Shift + P, the clipboard will be pause or resumed, but no notification will be shown. And as i remember, the app didnt ever show any other notifications. I just would like to see notification to be sure, that the clipboard is disabled. I need it when copying passwords, as i dont want them to be captured. And it would be more quick as clicking the tray icon every time.

this is weird, because on my macos and windows it works:-) I will check it again. Also, the app displays the notification each time you copy something on any other device and “Universal Clipboard” is activated. Please check it also, probably your systems blocks any notification globally.

**Comment by Senliast** [comment]
so i checked the notification settings for the app, on my Windows PC and MacBook both the notifications for the app was disabled. After i enabled it - the notifications for monitoring start / stop now appears just without problems! Lol thats strange, because i dont remember that i disabled it, no idea why it was disabled... Anyway now i know that the app already has the function and it works, thank you! The issue can be closed.

**Comment by Senliast** [comment]
edit: when notifications are enabled, also notifications about new clipboard contents will be shown. Could you add a switch to optionally disable them, so that only notifications about clipboard enabled / disabled will be shown?

**Comment by atrashler** [comment]
> edit: when notifications are enabled, also notifications about new clipboard contents will be shown. Could you add a switch to optionally disable them, so that only notifications about clipboard enabled / disabled will be shown?

the app displays notifications when the clipboard is changed by using “universal clipboard” (clips from other device). The app does not display notifications when the clipboard was changed by copying text on the same device. Could you confirm that you copy and see notifications on the same device.

**Comment by Senliast** [comment]
sorry, i mean exactly this, the notifications appears when clipboard changed from other device. And it would be great if you implement a toggle so that this notifications can be disabled.
